src/consensus/consensus_types.h

```diff
@@ -15,7 +15,7 @@ namespace consensus
   DECLARE_JSON_TYPE_WITH_OPTIONAL_FIELDS(Configuration);
   DECLARE_JSON_REQUIRED_FIELDS(Configuration);
   DECLARE_JSON_OPTIONAL_FIELDS(
-    Configuration, message_timeout, election_timeout);
+    Configuration, message_timeout, election_timeout, max_uncommitted_tx_count);
```

Comment:
Nothing to change here, but a note for posterity: the `OPTIONAL` here means we won't serialise these fields if they match the default, when converting a `Configuration` to JSON. That's a subtle and potentially confusing behaviour - nodes with different defaults may respond differently with a partial object, and wherever the config is reported it may be missing fields that need to be inferred from code defaults. That's already true of the other fields here, so the new field isn't special. I _think_ its safe because we don't reserialise this, but that's a guess.